tests/unit/api.py

```diff
@@ -1346,6 +1346,7 @@ def test_no_project_networks(self, db):
         api.project_create('anvil-nextgen')
         assert json.loads(api.list_project_nodes('anvil-nextgen')) == []
 
+    import uuid
```

Comment:
Why is this imported way down here, instead of at the top? Noticed this while merging back into my auth branch for #491.